components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/AccessTokenDAOImpl.java

```diff
@@ -549,7 +549,8 @@ private String getLatestAccessTokenQuerySQL(Connection connection) throws SQLExc
         if (OAuth2ServiceComponentHolder.isIDPIdColumnEnabled()) {
             if (OAuth2ServiceComponentHolder.isConsentedTokenColumnEnabled()) {
                 if (connection.getMetaData().getDriverName().contains("MySQL")
-                        || connection.getMetaData().getDriverName().contains("H2")) {
+                        || connection.getMetaData().getDriverName().contains("H2")
+                        || connection.getMetaData().getDriverName().contains("MariaDB")) {
```

Comment:
Isn't this string available as a constant anywhere ?

Reply:
There are no dedicated constants for db types in this repo and everywhere the db type is hardcoded. However we can use the constants from here https://github.com/wso2/carbon-identity-framework/blob/master/components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationConstants.java#L75

Reply:
Fixed it in https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2016/commits/2d75f072c8f705f650941b69ad445e8939a7acce